extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -190,6 +190,14 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     this.boundNoContentHandler_ = () => this.noContentHandler_();
 
     setupA2AListener(this.win);
+
+    /** @private @const {function()|null}*/
+    this.renderStartResolve_ = null;
+
+    /** @private @const {!Promise}*/
+    this.renderStartPromise_ = new Promise(resolve => {
```

Comment:
It's not used for now. One thing we can use it for is ask amp-sticky-ad to display after this promise is resolved.